x-pack/plugins/observability_solution/apm/public/hooks/use_eco_tour.tsx

```diff
@@ -7,11 +7,11 @@
 
 import { useLocalStorage } from './use_local_storage';
 
-type TourState = 'shownBefore' | 'isActive';
+type TourState = 'isModalVisible' | 'isTourActive';
 
-const INITIAL_STATE: Record<TourState, boolean> = {
-  shownBefore: false,
-  isActive: false,
+const INITIAL_STATE: Record<TourState, boolean | undefined> = {
+  isModalVisible: undefined,
```

Comment:
The initial state for the modal is undefined as it needs to be true when the new experience is really ready (EEM enabled)

it will false when the user will close the modal